best-practices/require-ro-rootfs/bad-pod-notall.yaml

```diff
@@ -22,4 +22,4 @@ spec:
     securityContext:
       readOnlyRootFilesystem: true
   - name: busybox
-    image: busybox:1.35
\ No newline at end of file
+    image: busybox:1.35
```

Comment:
If we're not materially changing code, we should not be submitting these sort of changes to a public repo.  It just leads to more noise when we merge from public upstream. Can this change be reverted?